flake.nix

```diff
@@ -113,15 +118,13 @@
 
             # Running setuptoolsShellHook by default is confusing because it
             # will fail if SimpLL hasn't been built before.
-            dontUseSetuptoolsShellHook = true;
```

Comment:
Also remove the above comment and adjust the comments below? IIUC, we're not using `setuptoolsShellHook` anymore, are we?

Reply:
We are not, I am going to update it.